go.mod

```diff
@@ -218,8 +218,6 @@ replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
 	// ics23 patch for dragonberry
 	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
-	// use Cosmos-SDK fork to enable Ledger functionality
-	github.com/cosmos/cosmos-sdk => github.com/evmos/cosmos-sdk v0.46.6-ledger
```

Comment:
undo, we need to use the fork for ledger

Reply:
we just need to update it to `v0.46.7-ledger`

Reply:
not yet available

Reply:
Yeah I realised this after pushing